### DISCUSSION TITLE
Stop Auto-Reconnect

### DISCUSSION THREAD

**Discussion by ArcticCascade** [pschatzmann/ESP32-A2DP#119] [Q&A] [answered]
Hi, I'm new to ESP32 and Bluetooth coding in general, so forgive me if this is a silly question.
I would like to use my ESP as a Bluetooth music receiver, but it is to be used by many people (one at a time) in a public place, so I need it to stop devices that have previously paired from automatically reconnecting to it.
I was hoping that simply changing the name would be good enough, but a quick test shows that my phone doesn't care what the name is. There must be some other identifier that I'm hoping to be able to randomize.
Can anyone tell me if this is possible?

**Comment by pschatzmann** [comment, marked as answer]
I suggest to have a look at the API documentation. In the start method you can switch the autoreconnect off...

https://pschatzmann.github.io/ESP32-A2DP/html/class_bluetooth_a2_d_p_sink.html

**Reply by ArcticCascade** [reply]
Thank you, but that's not exactly what I'm after. It still shows up as a known device (with a new name) in my phone's Bluetooth list. I'd like it to appear as a brand new device each time it starts up. I don't even know if this is possible, but I'm hoping it is!

**Comment by pschatzmann** [comment]
Each device has a Bluetooth device address (search for esp_bd_addr_t).  You could try to play around with the methods documented in https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-reference/bluetooth/esp_gap_bt.html

The esp_bt_gap_remove_bond_device() is maybe helpful...
